### ISSUE TITLE
Ensembling/ Stacking different pretrained weights for the classification. #GSSOC'22

### ISSUE THREAD

**Issue by hemhemoh** [srimani-programmer/Breast-Cancer-Predictor#201]
Hi, I recently raised an issue where I said I was going to use transfer learning for the dataset, which I have done and created a pull request for(VGG16). However, in the cause of my research, I found out that medical imaging is a field where ML is currently thriving, so my 0.947 validation accuracy might not be good enough as every 0.1 matters here. I would appreciate it if this issue is assigned to me as I would be training about 5 different pre-trained models and stacking up(ensembling) the best 3, if not all 5.

**Comment by freny24** [comment]
@hemhemoh issue has been assigned to you! Good Luck

**Comment by hemhemoh** [comment]
@freny24 please close this issue, the pull request has been merged
